app/services/idv/steps/in_person/address_step.rb

```diff
@@ -38,7 +41,7 @@ def capture_secondary_id_enabled?
           current_user.establishing_in_person_enrollment.capture_secondary_id_enabled
         end
 
-        def updating_address
+        def updating_address?
           flow_session[:pii_from_user].has_key?(:address1)
```

Comment:
Will this check work? I think the `flow_session[:pii_from_user]` object will always have an `address1` key because it gets copied over in the state id step. See [Gina's PR](https://github.com/18F/identity-idp/pull/8288/files#diff-a8c6060678cd6f49b47878fa15569f22c908a557d478f85aaa198ae3d3af6935R64)